recipe/meta.yaml

```diff
@@ -7,17 +7,17 @@ package:
 
 source:
   url: https://github.com/numpy/numpy/releases/download/v{{ version }}/numpy-{{ version }}.tar.gz
-  sha256: a3bccb70ad94091a5b9e2469fabd41ac877c140a6828c2022e35560a2ec0346c
+  sha256: 47b7b6145e7ba5918ce26be25999b6d4b35cf9fbfdf46b7da50090ffdb020445
   patches:
     # Apparently not all glibcs have 128 bit precision floats implemented correctly.
-    - skip_test_loss_of_precision.diff  # [aarch64 or ppc64le]
-    - skip_test_mem_overlap.diff        # [aarch64]
-    - skip_test_xerbla.diff             # [aarch64]
-    - skip_test_einsum_cfloat64.diff    # [ppc64le]
-    - skip_structured_object.diff       # [win and py27]
+    - patches/skip_test_loss_of_precision.diff  # [aarch64 or ppc64le]
+    - patches/skip_test_mem_overlap.diff        # [aarch64]
+    - patches/skip_test_xerbla.diff             # [aarch64]
+    - patches/skip_test_einsum_cfloat64.diff    # [ppc64le]
```

Comment:
These patches were not sent upstream? Why?

Reply:
They appear to be just skipping tests that don't work on the respect archs. So I'm not sure if it makes sense to upstream these patches per se.

Though it may be worthwhile to raise issues about these test failures (if that hasn't already happened). In any event these were already here before this PR.

Reply:
I believe that @hmaarrfk is the author of those patches. However, I'm not sure upstream is interested in patches that skips tests that they don't even run.

Reply:
I guess I never opened bug requests for numpy. 
I did for scipy.
These seem to be edge cases in aarch64.
It could be dependent on the underlying version of libc.

Reply:
Its also unclear if these are failures due to emulation or due to true bugs in numpy. Numpy does test for aarch64. so I was unsure what to do.

Reply:
mem_overlap might just be really slow. Testing numpy on my own computer, it runs SUPER slow, so on an emulator and aarch64, it could be wayyyyy to slow.

Reply:
https://github.com/conda-forge/numpy-feedstock/pull/157/files runs on native hardware

Reply:
Yeah, i believe that PR was coming to life at the same time as numpy aarch64 was being built.

I think the conclusion was that the only patch that was necessary there was
```
  patches:
    # I think this is because conda-forge supports an older GCC than what numpy tests with
    - skip_test_loss_of_precision.diff  # [aarch64]
```
and there is a comment in numpy that older GCC doesn't deal with with high precision floats.

Should we try building this on native and removing many patches?

Reply:
Yeah that might be a good idea.